### DISCUSSION TITLE
How to fetch data for alerts contained in alert groups

### DISCUSSION THREAD

**Discussion by sdarre** [Security-Onion-Solutions/securityonion#9911] [Unsupported Versions]
How can I fetch records from the Elastic store for alerts contained in a group using the search query and event filter? 

I want the groups to be displayed in different colors depending on data from the alerts contained in the group. However, the SO console does not load in any data for individual alerts when displaying groups. How can I achieve this?

Thank you.

**Comment by InfosecGoon** [comment]
I'm sorry, I'm having a little trouble understanding your question.

If you want to look at alert data in Hunt, you can do that with a query like this:

`event.dataset:"alert" | groupby -pie event.module
`
That will produce a pie graph with the different alerting subsystems highlighted in different colors.
